strider/fetcher.py

```diff
@@ -124,10 +124,17 @@ async def setup(
             self.preferred_prefixes,
         ))["query_graph"]
 
-        # Expand the query graph using the biolink model hierarchies
+        # Fill in missing categories and predicates using normalizer
         await fill_categories_predicates(self.qgraph, self.logger)
         standardize_graph_lists(self.qgraph)
 
+        # Replace biolink:Proten with biolink:GeneOrGeneProduct
+        for node in self.qgraph["nodes"].values():
+            categories = node.get("category", [])
+            for category in categories:
+                if category == "biolink:Protein":
+                    category = "biolink:GeneOrGeneProduct"
```

Comment:
I suspect that this does not actually edit the qgraph and that the test is not testing what you want it to test.